generate-code.sh

```diff
@@ -128,7 +138,49 @@ _generateData() {
       "${proto_src}"
 }
 
+_generateValidationTests() {
+    # Source the schema protos.
+    proto_dir="${GENERATE_DATA_SOURCE}/proto"
+    proto_src=$(realpath --relative-to="${proto_dir}" "$1")
+    # Derive path to data.proto from event.proto.
```

Comment:
How reliable is this assumption? is this naming enforced anywhere?

Reply:
data.proto and events.proto are consistently in the same directory. I'm not sure if this is enforced. @marcgovi is this a required or accidental structure?

Reply:
Discussed with @marcgovi, he says this is incidental but is looking into whether formalizing makes sense and plans to get back to us this week.

Reply:
Added a comment noting the consistent but non-standardized nature of event.proto location, will update once standards emerge.

Reply:
+1 to the comment added. Officially we do not have any enforcement in place today. I did some digging, and if I'm not mistaken @jskeet was the architect of this structure. The idea was to keep the `data.proto` as identical as possible as the API message proto.
In any case, I will create an AI on our end to setup some kind of enforcement around it to prevent pushing unwished structures.